Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class BalanceStats
+    {
+        private const string InValidOutput = "N/A.";
+        private StringBuilder _persons;
+        private decimal _amount;
+
+        public BalanceStats(StringBuilder persons, decimal amount)
+        {
+            _persons = persons;
+            _amount = amount;
+        }
+
+        public StringBuilder GetPersons()
+        {
+            return _persons; 
+        }
+
+        public decimal GetAmount()
+        {
+            return _amount; 
+        }
+
+        public static bool IsInValidInput(string[] input) => input == null || input.Length == 0;
+
+        public static string FindHighestBalanceEver(string[] peopleAndBalances)
+        {
+            if (IsInValidInput(peopleAndBalances))
+            {
+                return InValidOutput;
+            }
+
+            var answer = BalanceParser.FindHighestBalance(peopleAndBalances);
+
+            return $"{answer.GetPersons()} had the most money ever. ¤{answer.GetAmount()}.";
+        }
+
+        //public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
+        //{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)